### DISCUSSION TITLE
Add Usage Instructions

### DISCUSSION THREAD

**Discussion by jmoubry** [porrey/Virtual-ZPL-Printer#54] [Ideas]
I ran the installer and was able to get the active printer started, but I don't know how to make my applications recognize it as a ZPL printer. Are there additional steps?

**Comment by porrey** [comment]
This is designed for applications that communicate directly to the printer over a TCP/IP connection. In this use case, the client sends a ZPL string to the printer.

**Comment by alexbass16** [comment]
@porrey  thanks for your work! Could you please help me understand whether it is possible to use your App in conjunction with the Hematrax Client/Server toolset? https://www.digi-trax.com/wp-content/uploads/2019/05/Release-notes-HTUnity_14050.0915.pdf
I'm struggling to pair my system under the test which performs API call to hematrax server, and then hematrax sends data to zebra printer. 
I added new printer via Zebra Utiilities and it forwards test page to Virtual Printer without any issues. However, when I'm using "Print" button from Hematrax Client, I'm receiving "The Line Printer Daemon (LPD) service could not satisfy the request from 192.168.1.206, most likely because of network problems. Check network connectivity between the two computers." in Event Viewer. 
Any thoughts are highly appreciated, and thanks one more time

**Comment by porrey** [comment]
This app implements a raw TCP connection and not the LPD protocol.
